pwiz_tools/Skyline/Model/Serialization/DocumentFormat.cs

```diff
@@ -69,15 +69,18 @@ public struct DocumentFormat : IComparable<DocumentFormat>
         public static readonly DocumentFormat VERSION_4_13 = new DocumentFormat(4.13); // Adds new audit log format
         public static readonly DocumentFormat VERSION_4_2 = new DocumentFormat(4.2); // Release format
         public static readonly DocumentFormat VERSION_4_21 = new DocumentFormat(4.21); // Adds Lists feature
-        public static readonly DocumentFormat VERSION_4_22 = new DocumentFormat(4.22); // Moves explicite CE, explicit ion mobility high energy offset etc to transition instead of peptide level
+        public static readonly DocumentFormat VERSION_4_22 = new DocumentFormat(4.22); // Moves explicit CE, explicit ion mobility high energy offset etc to transition instead of peptide level
         public static readonly DocumentFormat VERSION_19_1 = new DocumentFormat(19.1); // Release format
         public static readonly DocumentFormat VERSION_19_11 = new DocumentFormat(19.11); // Annotation expressions
         public static readonly DocumentFormat VERSION_19_12 = new DocumentFormat(19.12); // Adds sample_id and serial_number
         public static readonly DocumentFormat VERSION_20_1 = new DocumentFormat(20.1); // Release format
         public static readonly DocumentFormat VERSION_20_11 = new DocumentFormat(20.11);
         public static readonly DocumentFormat VERSION_20_12 = new DocumentFormat(20.12); // Crosslinked peptides
         public static readonly DocumentFormat VERSION_20_13 = new DocumentFormat(20.13); // Add decoy_match_proportion
-        public static readonly DocumentFormat CURRENT = VERSION_20_13;
+        public static readonly DocumentFormat VERSION_20_14 = new DocumentFormat(20.14);
+        public static readonly DocumentFormat CURRENT = VERSION_20_14;
+        public static readonly DocumentFormat TRANSITION_SETTINGS_ION_MOBILITY = CURRENT; // First version with ion mobility settings moved from PeptideSettings to TransitionSettings

```

Comment:
This should be:
TRANSITION_SETTINGS_ION_MOBILITY = VERSION_20_14;

Reply:
Agreed that this needs to be set in stone at actual merge time, will do.

Are we also agreed that "TRANSITION_SETTINGS_ION_MOBILITY = CURRENT" is how it should be set during branch development?  

I imagine we could work out a git commit hook that forbids "= CURRENT" in DocumentFormat.cs on master but allows it on branches.

Reply:
No. It should be TRANSITION_SETTINGS_ION_MOBILITY = VERSION_20_14 right now and at merge time, if you find someone else has taken 20.14, then you to add 20.15 and set your constant to that. This seems like the same issue you were worrying about originally. I don't get why this is such a challenging concept, but I assume it will eventually make as much sense as I think it does to me and Nick. Or maybe we are missing some key issue?

Reply:
I'm sure I'm just over thinking it.

The only thing I still feel a little wiggly about is how we make sure that the next Daily goes out with a file actually called skyline_20.14.xsd, but I guess that's just something Brendan adds to his release procedures checklist.

Reply:
We don't currently release our .xsd files in any way other than having them available in version control for our open source project, and no one has ever asked for one, other than to consult for Panorama upload. We only do our file validation against them during testing. Then we need to have the named versions to validate against versions created or used in our tests, I think. Though, it may also be that we only validate documents persisted during the test, in which case we would only need skyline_current.xsd. Unless we are testing File > Share to older versions.

@nickshulman, you have looked most closely at this. Do the tests actually use the older schema versions, or is this our old-fashioned way of tracking versioning, which could now better be done with Git tags?

Reply:
We do have backward compatibility tests that depend on the existence of these suitably named xsd files, yes.